server/tests/audit_math/test_bravo.py

```diff
@@ -395,6 +395,10 @@ def test_compute_risk(contests):
             ("Biden", "Overvote"): 0.0,
             ("Biden", "Undervote/Blank"): 0.0,
         },
+        "test_caroline": {
```

Comment:
As much as I love calling this the Caro test 😆, a name that describes the situation might be more helpful down the road